Payload/Generator.php

```diff
@@ -139,11 +92,11 @@ protected function getRequestInfo(): array
         ];
     }
 
-    /**
-     * Get server info.
-     *
-     * @return array
-     */
+    #[ArrayShape(['host' => "false|string",
+        'root' => "string",
+        'user' => "string",
+        'file' => "mixed|null",
+        'argv' => "string"])]
     protected function getServerInfo(): array
```

Comment:
I'm getting an error [here](https://github.com/rollbar/rollbar-php-symfony-bundle/blob/261a0a5c0d794b9c7e23e47a37746dc039b46ec2/Payload/Generator.php#L107) saying that Kernel has no method `getRootDir`. In the composer file we specify `symfony/http-kernel: ^5.4|^6.0` and [http-kernel CHANGELOG](https://github.com/symfony/http-kernel/blob/94986633e4c3e7facb7defbd094a2e1170486ab5/CHANGELOG.md#500) says they removed that method.

Reply:
Maybe what we should use here is `$kernel->getProjectDir()`?

Reply:
It looks like the reason this was not failing in CI is because we have the `getRootDir()` method defined in our test `AppKernel` fixture.

Reply:
This has been updated.